packages/apify/src/actor.ts

```diff
@@ -646,7 +648,16 @@ export class Actor<Data extends Dictionary = Dictionary> {
      * @ignore
      */
     async getInput<T = Dictionary | string | Buffer>(): Promise<T | null> {
-        return this.getValue<T>(this.config.get('inputKey'));
+        const inputSecretsPrivateKeyFile = this.config.get('inputSecretsPrivateKeyFile');
+        const input = await this.getValue<T>(this.config.get('inputKey'));
+        if (inputSecretsPrivateKeyFile && ow.isValid(input, ow.object.nonEmpty) && inputSecretsPrivateKeyFile) {
```

Comment:
`inputSecretsPrivateKeyFile` is here twice

Reply:
But maybe you should check here for `inputSecretsPrivateKeyPassphrase` instead, it could happen that someone would pass only the secrets file but not the passphrase, and then it would break.